### PR TITLE
Add gitpod as development option

### PR DESCRIPTION
**Pull request by Furisto**
@utam0k Do you have permissions to install the [app](https://github.com/apps/gitpod-io) for this repository? Then we could also enable prebuilds.

<a href="https://gitpod.io/#https://github.com/containers/youki/pull/576"></a>

### CODE REVIEW

**Comment by utam0k** [comment]
@Furisto I've tried. What do you think?

**Comment by Furisto** [comment]
@utam0k Ok, seems to be working.

**Comment by utam0k** [comment]
> @utam0k Ok, seems to be working.

@Furisto 
Can I do the tutorial which be written
in README?
I have tried this a few month ago and failed😢

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+  - name: Build
+    init: cargo build 
+github:
+  prebuilds:
+    pullRequestsFromForks: true
+    addBadge: true
```
Comment (utam0k) [inline .gitpod.yml:9]: How about adding the `rust-analyzer` extension?

**Comment by utam0k** [comment]
Maybe we need to use `qemu` to run the tutorial, but gitpod is enough helpful to review PR and so on.

**Review by utam0k — APPROVED** [review]
This is enough useful to review PR or something. Looks good.

**Pull request merged**